### PR TITLE
adding the conda recipes for inmoose tool

### PR DESCRIPTION
**Pull request by EngyNasr**
Dear All, 

It's my first bioconda recipe, so I believe that this will be followed up with comments and updates. 

The tool name is [Inmoose](https://github.com/epigenelabs/inmoose/tree/master), it is mainly designed for batch correction of transcriptomic data, however it can also perform further analysis such as differential expression. I need this tool in one of my workflows created on Galaxy platform. 

Thanks a lot,
Engy

### CODE REVIEW

File: recipes/inmoose/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{% set name = "inmoose" %}
+{% set version = "0.7.2" %}
+{% set md5 = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" %}
```
Comment (martin-g) [inline recipes/inmoose/meta.yaml:3]: This looks like `sha256`, not `md5`.
Please use `sha256`!

File: recipes/inmoose/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/epigenelabs/inmoose/archive/refs/tags/v0.7.2.tar.gz
```
Comment (martin-g) [inline recipes/inmoose/meta.yaml:10]:
```suggestion
  url: https://github.com/epigenelabs/inmoose/archive/refs/tags/v{{ version }}.tar.gz
```

File: recipes/inmoose/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  md5: {{ md5 }}
```
Comment (martin-g) [inline recipes/inmoose/meta.yaml:11]:
```suggestion
  sha256: {{ md5 }}
```

File: recipes/inmoose/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+build:
+  noarch: python
+  number: 0
```
Comment (martin-g) [inline recipes/inmoose/meta.yaml:15]:
```suggestion
  number: 0
  run_exports:
    - {{ pin_subpackage(name, max_pin="x.x") }}
```

File: recipes/inmoose/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    - consensusClustering
+    - sklearn.cluster
+    - AgglomerativeClustering
+
+  commands:
+    - inmoose --help
+
+about:
+  home: https://github.com/epigenelabs/inmoose/tree/master
+  license: GPL-3.0 license
```
Comment (martin-g) [inline recipes/inmoose/meta.yaml:57]:
```suggestion
  license: GPL-3.0-or-later
  license_family: GPL3
  license_file: LICENSE
```

File: recipes/inmoose/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - pandas
+    - seaborn
+    - matplotlib
```
Comment (martin-g) [inline recipes/inmoose/meta.yaml:32]:
```suggestion
    - matplotlib-base
```

File: recipes/inmoose/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+  run_exports:
+  - {{ pin_subpackage(name, max_pin="x.x") }}
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  entry_points:
+    - inmoose = inmoose.__master__:root_command
+
+requirements:
+  host:
+    - python
+    - pip
```
Comment (martin-g) [inline recipes/inmoose/meta.yaml:25]:
```suggestion
    - pip
    - numpy
```

**Review by mencian — APPROVED** [review]

**Comment by EngyNasr** [comment]
Thanks a lot @mencian :)

**Pull request merged**